### ISSUE TITLE
Node Alias

### ISSUE THREAD

**Issue by Zehir** [cakephp/acl#116]
Hello, In this file;

File: [src/Model/Behavior/AclBehavior.php](https://github.com/cakephp/acl/blob/master/src/Model/Behavior/AclBehavior.php#L142) L142:
``` php
if (method_exists($entity, 'nodeAlias')) {
    $data['alias'] = $entity->nodeAlias;
}
```
Why he check if method exists and then get the value of property?
To make this work I put this code in method parentNode to initialize the value:
``` php
$this->nodeAlias = $this->nodeAlias();
``` 
Did I miss something?

**Comment by rchavik** [comment]
@dakota was this a typo?

**Comment by Zehir** [comment]
Commit reference: b96181240cc53507e3430f596d7ad95a910ce384

**Comment by dakota** [comment]
Oh dear, that is definately a typo.

**Comment by dakota** [comment]
Closed with #118
